src/core/client_config/resolvers/dns_resolver.c

```diff
@@ -42,8 +42,14 @@
 #include "src/core/client_config/lb_policy_registry.h"
 #include "src/core/iomgr/resolve_address.h"
 #include "src/core/iomgr/timer.h"
+#include "src/core/support/backoff.h"
 #include "src/core/support/string.h"
 
+#define BACKOFF_MULTIPLIER 1.6
```

Comment:
in subchannel.c, the corresponding defines all have a GRPC_SUBCHANNEL_ prefix. Add the corresponding GRPC_DNS_RESOLVER_ one here for consistency?